lib/cli/generators/ANGULAR/angular-helpers.js

```diff
@@ -35,3 +35,8 @@ export function editStorybookTsConfig(tsconfigPath) {
   tsConfigJson = setStorybookTsconfigExtendsPath(tsConfigJson);
   writeFileAsJson(tsconfigPath, tsConfigJson);
 }
+
+export function checkForDefaultProjectExist() {
```

Comment:
Maybe `ngDefaultProjectExists()` or something like that?
Reads like `if( ngDefaultProjectExists() )` instead of `if( checkForDefaultProjectExist() )`

Reply:
@kroeder `isDefaultProjectPresented` may be?

Reply:
Why not! More technical name would be `isDefaultProjectSet`
Choose what you prefer 👍

Reply:
`isDefaultProjectSet`

Reply:
done :)